developer-docs-site/docs/tutorials/run-a-fullnode.md

```diff
@@ -81,11 +81,11 @@ During the initial synchronization of your FullNode, there may be a lot of data
   fullnode_1  | INFO 2020-09-28T23:16:04.508902Z execution/executor/src/lib.rs:580 sync_finished {"committed_with_ledger_info":false,"name":"chunk_executor","synced_to_version":634000}
   ```
 
-* At the same time, the StateSync component will output similar information but show the destination.
+* At the same time, the state sync component will output similar information.
 
-* The number of connections should be more than 0, `curl 127.0.0.1:9101/metrics 2> /dev/null | grep aptos_connections`. If it's not, see the next section for potential fix.
+* The number of connections should be more than 0, `curl 127.0.0.1:9101/metrics 2> /dev/null | grep aptos_connections`. If it's not, see the next section for potential solutions.
```

Comment:
maybe here we can specify that they should check the `outbound ` metrics, I think aptos_connections has both outbound and inbound, people might have inbound connection as 0

Reply:
Yeah -- makes sense! I'll follow up with some of my other fixes, too 😄 Thanks @sherry-x!